README.md

```diff
@@ -61,6 +61,33 @@ To launch the complete test matrix run:
     $ tox
 
 
+#### Running Tests in docker
+
+If you prefer not to setup your local machine to run tests, we provide a preconfigured docker image.
+Note that this image is the same used in CircleCI to run tests.
+
+You still need docker containers running additional services up and running.
+
+Run the test runner
+
+    $ docker-compose run --rm testrunner
+
+Now you are in a bash shell. You can now run tests as you would do in your local environment:
+
+    $ tox -e '{py35,py36}-redis{210}'
+
+If you are in a unix machine, we also provide a shell script to execute commands in the provided container (so you don't
+forget to remove-`--rm` the container after you run it).
+
+For example to run the tests for `redis-py` 2.10 on Python 3.5 and 3.6:
+
+    $ ./scripts/ddtest tox -e '{py35,py36}-redis{210}'
+
+You can also add the `scripts` folder to your path, so then you can run
```

Comment:
This last part we can remove.

I think if we want to keep this suggestion we need to add instructions on how to add it to your path, and then we need to update the script to be runnable from any location on your system.

e.g. if I added to my path and did:

```bash
cd ~/Desktop
ddtest bash
```

I'd get a `docker-compose` error since the script will try to run `docker-compose` in my current directory.

I've done it before where it can be runnable from anywhere so can provide guidance there if that is the direction we want to head in.